defaults/main.yml

```diff
@@ -15,3 +15,15 @@ list_of_dict_sa_with_roles:
   - { sa_name: "{{ autobot_jenkins_sa }}", role: 'edit' }
   - { sa_name: "{{ koji_sa }}", role: 'edit' }
   - { sa_name: "{{ ansible_tower_sa }}", role: 'admin' }
+
+allowed_domains:
```

Comment:
remove all of those, you can add : example......

Reply:
Removed :)